contrib/alertmanager.js

```diff
@@ -0,0 +1,47 @@
+function statusBadge(status, severity) {
```

Comment:
I'd:

- Put this in a subdirectory like jsTransformationFunctions
- Add a simple description of what this does in docstring.